[IMP] ecommerce: updated catalog page (#2971)
odoo/documentation: content/applications/websites/ecommerce/managing_products/catalog.rst

============================
Organize the product catalog
============================

The eCommerce catalog is the equivalent of your physical store shelves: it allows customers to see
what you have to offer. Clear categories, available options, sorting, and navigation threads help
you structure it efficiently.

Categorize the product catalog
==============================

In Odoo, there is a **specific category model** for your eCommerce. Using eCommerce categories for
your products allows you to add a navigation menu on your eCommerce page. Visitors can then use it

> to view all products under the category they select. To do so, go to :menuselection:`eCommerce -->

Reviewer: ```suggestion
to view all products under the category they select. To do so, go to :menuselection:`Website app --> eCommerce -->
```

Products --> (Your product) --> Sales` and select the :guilabel:`Categories` you want under

> :guilabel:`eCommerce Shop`.

Reviewer: I would replace it by something like:
Go to :menuselection:`Website app --> eCommerce`, select a product, go to the :guilabel:`Sales` tab and Sales and under the eCommerce Shop, add the categories you want.

.. note::
   A single product can appear under multiple eCommerce categories.

> When your product is configured, go to the :guilabel:`Shop` page of your website and click on

Reviewer: what do you mean by configured? maybe clarify

> :menuselection:`Edit --> Customize`. In the :guilabel:`Categories` option, you can either enable a

Reviewer: :menuselection:`Edit --> Customize tab`

menu on the :guilabel:`Left`, on the :guilabel:`Top`, or both. The :guilabel:`Collapsable Category

> Recursive` button renders the *left* category menu collapsable.

Reviewer: If you select the :guilabel:`Left` category, the option :guilabel:`Collapsable Category Recursive` appears and it allows to render the :guilabel:`Left` category menu collapsable.

.. image:: catalog/catalog-panel-categories.png
   :align: center
   :alt: Categories options for your eCommerce website

Browsing
--------

The eCommerce category is the first tool to organize and split your products. However, if you need
an extra level of categorization in your catalog, you can activate various **filters** such as
attributes or sort-by search.

> Attributes

Reviewer: Attributes and variants?

Author: There's a :seealso: just below

~~~~~~~~~~

Attributes refer to **characteristics** of a product such as **color** or **material**, whereas

> variants are the different combinations of attributes. Attributes and variants can be found under

Reviewer: :guilabel:`Attributes and variants` can be found under

> :menuselection:`eCommerce --> Products --> (Your product) --> Attributes & Variants`.

Reviewer: :menuselection:`Website app --> eCommerce --> Products` select your product and go to the  :guilabel:`Attributes & Variants` tab.

.. seealso::
   - :doc:`../../../sales/sales/products_prices/products/variants`

.. image:: catalog/catalog-attributes.png
   :align: center
   :alt: Attributes and variants of your product

> To enable **attributess** filtering, go to :menuselection:`Shop page --> Edit --> Customize` and

Reviewer: **attributess** typo

select either :guilabel:`Left`, :guilabel:`Top`, or both. Additionally, you can also enable
:guilabel:`Price Filtering` to enable price filters.

.. note::
   :guilabel:`Price Filter` works independently from **attributes** and therefore can be enabled on
   its own if desired.

.. tip::
   You can use **attribute filters** even if you do not work with product variants. When adding
   attributes to your products, make sure only to specify *one* value per attribute. Odoo will not
   create variants if no combination is possible.

Sort-by search
~~~~~~~~~~~~~~

It is possible to allow the user to manually **sort the catalog** using the search bar. From

> :menuselection:`Shop page --> Edit --> Customize`, you can enable or disable the :guilabel:`Sort-By`

Reviewer: go to :menuselection:`Wesbite app --> Shop` click on :guilabel:`Edit` and then to the :guilabel:`Customize` tab

option as well as the :guilabel:`Layout` button. You can also select the :guilabel:`Default Sort` of
the :guilabel:`Sort-By` button. The default sort applies to *all* categories.

The **sorting** options are:

- Featured
- Newest Arrivals
- Name (A-Z)
- Price - Low to High
- Price - High to Low

In addition, you can **manually edit** the order of a product by going to :menuselection:`Shop page

> --> Edit` and clicking on the product. Under the :guilabel:`Product` section of the

Reviewer: go to :menuselection:Wesbite app --> Shop click on :guilabel:Edit and then select the product.

Reviewer: please also update for the other paths below :)

:guilabel:`Customize` section, you can rearrange the order by clicking on the arrows. `<<` `>>` move
the product to **extreme** right or left, and `<` `>` move the product by **one** row to the right
or left.

.. image:: catalog/catalog-reorder.png
   :align: center
   :alt: Product rearrangement in the catalog

Page design
===========

Category page
-------------

You can customize the layout of the category page using the website builder. Note that editing the
layout of the category page is global; editing one category layout will affect *all* category pages.

To do so, go on of your :menuselection:`Category page --> Edit --> Customize`. Here, you can choose
the layout, the number of columns to display the products, etc. The :guilabel:`Product Description`
button makes the product description visible from the category page, underneath the product picture.

> .. image:: catalog/catalog-category-layout.png

Reviewer: Would it be possible to resize the image? It looks big...

=======
Catalog
=======

The eCommerce catalog is the equivalent of your physical store shelves: it allows customers to see
what you have to offer. Clear categories, available options, sorting, and navigation threads help
you structure it efficiently.

Categorize the product catalog
==============================

In Odoo, there is a **specific category model** for your eCommerce. Using eCommerce categories for
your products allows you to add a navigation menu on your eCommerce page. Visitors can then use it

> to view all products under the category they select. To do so, go to :menuselection:`Website app

Reviewer 2: Suggestion: 
split before "to do so" (it could be in a separate paragraph anyway) and add an image to illustrate (not too large)?

> to view all products under the category they select. To do so, go to :menuselection:`Website app

Reviewer 2: no need to write app everywhere
```suggestion
to view all products under the category they select. To do so, go to :menuselection:`Website 
```

Reviewer 2: same below

--> eCommerce --> Products`, select the product you wish to modify, click on the :guilabel:`Sales`
tab. and select the :guilabel:`Categories` you want under :guilabel:`eCommerce Shop`.

.. note::
   A single product can appear under multiple eCommerce categories.

When your product's categories are configured, go to :menuselection:`Website dashboard --> Shop` and

> click on :menuselection:`Edit --> Customize tab`. In the :guilabel:`Categories` option, you can

Reviewer 2: There is no website dashboard from Odoo 16. Also, "shop" could be named differently, or placed differently in the top menu than how it is by default with the demo data.
I suggest
"Go to your shop page" or something like that

Reviewer 2: same thing below

Reviewer 2: When I read the other iterations in this doc, I think the easiest is to write
"Start editing your [main?] shop page and open the xxx tab"

Author: Changed it all

either enable a menu on the :guilabel:`Left`, on the :guilabel:`Top`, or both. If you select the
:guilabel:`Left` category, the option :guilabel:`Collapsable Category Recursive` appears and allows
to render the :guilabel:`Left` category menu collapsable.

.. image:: catalog/catalog-panel-categories.png
   :align: center
   :alt: Categories options for your eCommerce website

Browsing
--------

The eCommerce category is the first tool to organize and split your products. However, if you need
an extra level of categorization in your catalog, you can activate various **filters** such as
attributes or sort-by search.

Attributes
~~~~~~~~~~

Attributes refer to **characteristics** of a product such as **color** or **material**, whereas
variants are the different combinations of attributes. :guilabel:`Attributes and Variants` can be
found under :menuselection:`Website app --> eCommerce --> Products`, select your product, and
:guilabel:`Attributes & Variants` tab.

.. seealso::
   - :doc:`../../../sales/sales/products_prices/products/variants`

.. image:: catalog/catalog-attributes.png
   :align: center
   :alt: Attributes and variants of your product

> To enable **attributess** filtering, go to :menuselection:`Website dashboard --> Shop`, click on

Reviewer 2: ```suggestion
To enable **attributes** filtering, go to :menuselection:`Website dashboard --> Shop`, click on
```

:menuselection:`Edit --> Customize tab` and select either :guilabel:`Left`, :guilabel:`Top`, or
both. Additionally, you can also enable :guilabel:`Price Filtering` to enable price filters.

.. note::
   :guilabel:`Price Filter` works independently from **attributes** and therefore can be enabled on
   its own if desired.

.. tip::
   You can use **attribute filters** even if you do not work with product variants. When adding

>    attributes to your products, make sure only to specify *one* value per attribute. Odoo will not
>    create variants if no combination is possible.

Reviewer 2: ```suggestion
   attributes to your products, make sure only to specify *one* value per attribute. Odoo does not
   create variants if no combination is possible.
```

can enable or disable the :guilabel:`Sort-By` option as well as the :guilabel:`Layout` button. You
can also select the :guilabel:`Default Sort` of the :guilabel:`Sort-By` button. The default sort
applies to *all* categories.

The **sorting** options are:

- Featured
- Newest Arrivals
- Name (A-Z)
- Price - Low to High
- Price - High to Low

In addition, you can **manually edit** the catalog's order of a products by going to
:menuselection:`Website dashboard --> Shop page --> Edit` and clicking on the product. Under the
:guilabel:`Product` section of the :guilabel:`Customize` section, you can rearrange the order by
clicking on the arrows. `<<` `>>` move the product to **extreme** right or left, and `<` `>` move
the product by **one** row to the right or left. It is also possible to change the catalog's order
of products in :menuselection:`Website app --> eCommerce --> Products` and dragging-and-dropping the
products within the list.

.. image:: catalog/catalog-reorder.png
   :align: center
   :alt: Product rearrangement in the catalog

Page design
===========

Category page
-------------

> You can customize the layout of the category page using the website builder. Note that editing the
> layout of the category page is global; editing one category layout will affect *all* category pages.

Reviewer 2: - use admonitions for that kind of comments (you even wrote "note that...")
- slightly rephrased (+ present tense)

```suggestion
You can customize the layout of the category page using the website builder. 

.. important::
   Note that editing the category page layout is global; editing one category layout affects *all* category pages.
```

To do so, go on of your :menuselection:`Category page --> Edit --> Customize`. Here, you can choose
the layout, the number of columns to display the products, etc. The :guilabel:`Product Description`
button makes the product description visible from the category page, underneath the product picture.

.. image:: catalog/catalog-category-layout.png
   :align: center
   :alt: Layout options of the category pages.

.. tip::
   You can choose the size of the grid, but be aware that displaying too many products may affect
   performances and page loading speed.

Product highlight
-----------------

You can highlight products to make them more visible on the category or product page. On the page of
your choice, go to :menuselection:`Edit --> Customize` and click on the product to highlight. In the
:guilabel:`Product` section, you can choose the size of the product image by clicking on the grid,
and you can also add a **ribbon** or :guilabel:`Badge`. This displays a banner across the product's
image, such as:

- Sale;
- Sold out;
- Out of stock;
- New.

> Alternatively, you can enable the :doc:`Debug mode <../../../general/developer_mode>` on the

Reviewer 2: it's named "developer mode", which can be activated ("debug mode" is a remnant of older times)
```suggestion
Alternatively, you can activate the :ref:`developer mode <developer-mode>` on the
```

Reviewer 2: same below

Author: Changed

or a **wishlist**. To do so, go to :menuselection:`Website dashboard --> Shop --> Edit -->
Customize`, and at the end of the :guilabel:`Products Page` category, click on the feature buttons
you wish to use. All three buttons appear when hoovering the mouse over a product's image.

- :guilabel:`Add to Cart`: adds a button to **add to cart** the product;
- :guilabel:`Comparison List`: adds a button to **compare** products based on their price, variant,
  etc;
- :guilabel:`Wishlist Button`: adds a button to **wishlist** the product.

.. image:: catalog/catalog-buttons.png
   :align: center
   :alt: Feature buttons for add to cart, comparison list, and wishlist

.. image:: catalog/catalog-features.png
   :align: center
   :alt: Appearance of buttons when hoovering over the mouse

Add content
===========

You can use **building blocks** to add content on the category page, with a variety of blocks
ranging from :guilabel:`Structure` to :guilabel:`Dynamic Content`. Specific areas are defined to use
blocks are defined and highlighted on the page when **dragging-and-dropping** a block.

.. image:: catalog/catalog-content.png
   :align: center
   :alt: Building blocks areas

> - If you drop a building block **on top** of the product list, it will create a new category header
>   specific to *that* category.
> - If you drop a building **on the top** or **bottom** of the page, it will be visible on *all*
>   category pages.

Reviewer 2: ```suggestion
- If you drop a building block **on top** of the product list, it creates a new category header
  specific to *that* category.
- If you drop a building **on the top** or **bottom** of the page, it is visible on *all*
  category pages.
```

=======
Catalog
=======

The eCommerce catalog is the equivalent of your physical store shelves: it allows customers to see
what you have to offer. Clear categories, available options, sorting, and navigation threads help
you structure it efficiently.

Categorize the product catalog
==============================

In Odoo, there is a **specific category model** for your eCommerce. Using eCommerce categories for
your products allows you to add a navigation menu on your eCommerce page. Visitors can then use it
to view all products under the category they select. To do so, go to :menuselection:`Website app
--> eCommerce --> Products`, select the product you wish to modify, click on the :guilabel:`Sales`
tab. and select the :guilabel:`Categories` you want under :guilabel:`eCommerce Shop`.

.. note::
   A single product can appear under multiple eCommerce categories.

> When your product's categories are configured, go to :menuselection:`Website app --> Site -->
> Homepage --> Shop` and click on :menuselection:`Edit --> Customize tab`. In the

Reviewer: ```suggestion
When your product's categories are configured, go to :menuselection:`Website dashboard --> Shop` and click on :menuselection:`Edit --> Customize tab`. In the
```
My suggestion is to take the fastest path and skip 2 steps, but up to you to follow it, or not.
This logic could be applied to all similar paths.

Author: Noted
